meetings/2025-09-24.md

```diff
@@ -0,0 +1,211 @@
+* SC: Please unarchive repositories that were archived without CG's consent, e.g., https://github.com/solid/test-suite-panel/
+* Jesse: I can provide context. We at the ODI are going through a cleanup of github.org/solid to do a better job of going through issue backlogs. We posted a spreadsheet open for comment for over two weeks and only enacted prescribed actions on things that were not commented on. Sarven then requested things be moved back. I'd like to discuss whether some work items can be considered not active and moved to solid-contrib.
+* eP: The least friction approach, I really appreciate the work to cleanup and groom GitHub. It is unfortunate Sarven didn't manage to be here today. solid-contrib is a bit unclear whether it is under ODI perview. We could archive things.
+* Jesse: We will moderate solid-contrib and add the code of conduct. We're not saying we won't ever touch a repo there, it's just that we're not actively monitoring that space. I'd like to check whether archiving is ok. We need a CG consensus to confirm the CG is ok with archiving.
+* Michiel: I value the work Jesse is doing here and we should let him move and archive as makes sense. Whatever is clearer to readers.
+* Jesse: I looked at the issue and Sarven asked to unarchive.
+* Michal: I didn't understand what Sarven said. I'm not sure if solid and solid-contrib belongs to CG or ODI?
+* Jesse: ODI has legal ownership of solid GitHub org and responsibility to monitor those repos. The only one deffered to the CG is solid/specification
+* eP: I found the quote I looked for:
+    * SC: If there is a proposal to archive repos: shapes, test-suite-panel, authentication-panel, solid-wg-charter, I have no objection. However, do not move them to another org or rename. 
+* eP: Just to avoid the debate we can archive things, instead of moving.
+* Jesse: For archiving, I am comfortable moving things back and archiving.
+
+### Ownership Question
+
+* Michal: I don't understand the meaning of legal ownership (especially for solid-contrib and practitioners things).
+* Jesse: We are moderating all repository on the solid and solid-contrib GitHub org and all codebases are MIT license.
+* Michiel: We can think of it as stewardship of these repos rather than ownership.
+* eP: Can we update the readme of both solid and solid-contrib GitHub orgs to have those details (stewardship/code of conduct).
+* Jesse: If an org like ODI is not moderating solid-contrib, we will hit issues where something such as solid server are not moderated. From a pragmatic standpoint, we just want consistent moderation of "official"/"popular" repos.
+* Michal: I'm just thinking from the point of view of moving repos and admin rights.
+* Jesse: I will come to the practitioners group to continue the conversation.
+* eP: It should be transparent in the readme of the organisations who is admin of the orgs. That will help.
```

Comment:
I'm not sure about blasting ODI everywhere. They are the admins on `solid` org for example but I don't think it makes any sense that something like `solid/specification` needs to mention ODI. The README there is already clear that it is W3C Solid CG workspace and the W3C Conduct etc. For other non-CG repos under `solid` or elsewhere, I agree with you.